modules/schema/src/main/scala/lucuma/odb/json/sourceprofile.scala

```diff
@@ -58,38 +58,6 @@ trait SourceProfileCodec {
   import numeric.given
   import wavelength.decoder.given
 
-  // SpectralDefinition[T]
-  given [T](using Enumerated[Units Of Brightness[T]], Enumerated[Units Of LineFlux[T]], Enumerated[Units Of FluxDensityContinuum[T]]): Decoder[SpectralDefinition[T]] =
```

Comment:
Reordering required by Scala 3.4.  Apparently a `given` definition must appear in the file before it is referenced elsewhere.